### PR TITLE
[collect tests] Collect for install - all XSIAM content items

### PR DESCRIPTION
**Pull request by mmhw**
## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
Fixes: [link to the issue](https://jira-hq.paloaltonetworks.local/browse/CIAC-5488)

## Description
The changes are except for the modeling rules, which will also be uploaded to the branch and nightly builds.

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [x] No

## Must have
- [x] Tests
- [ ] Documentation

### CODE REVIEW

File: Tests/scripts/collect_tests/constants.py
Comment (bziser) [inline Tests/scripts/collect_tests/constants.py:138]:
```suggestion
    FileType.CORRELATION_RULE,
```

File: Tests/scripts/infrastructure_tests/test_collect_tests.py
Comment (dorschw) [inline Tests/scripts/infrastructure_tests/test_collect_tests.py:240]: please document the case, here and at the top of the file (row 50)

**Review by bziser — APPROVED** [review]
great
wait for @dorschw 's approval

**Pull request merged**
